chia/server/outbound_message.py

```diff
@@ -1,22 +1,14 @@
 from __future__ import annotations
 
 from dataclasses import dataclass
-from enum import IntEnum
 from typing import Optional, SupportsBytes, Union
 
 from chia.protocols.protocol_message_types import ProtocolMessageTypes
+from chia.types.node_type import NodeType  # TODO: remove this and change a lot of `import` statements
```

Comment:
i believe this will indicate an explicit re-export to mypy so we don't have to maintain a `__all__`.

```suggestion
from chia.types.node_type import NodeType as NodeType  # TODO: remove this and change a lot of `import` statements
```

Reply:
Unfortunately this change is giving me `chia/server/outbound_message.py:7:34: PLC0414 Import alias does not rename original package`

Reply:
this is already a thing we intend to correct (per the `TODO:`) so i think an ignore is reasonable here.

```suggestion
# TODO: remove this and change a lot of `import` statements
from chia.types.node_type import NodeType as NodeType # noqa: PLC0414
```

i just really hate maintaining `__all__`, which won't generally happen i don't think unless some check catches it being out of date.  and i despise even more when it is leveraged (`*` imports).

for example, i did:

https://github.com/Chia-Network/chia_rs/blob/f512dee8c880446faa96972bb78b9c65c250af13/wheel/python/chia_rs/datalayer.pyi#L96-L97
```python
# just disallow * importing so we don't have to maintain this repetitive list
__all__: Sequence[str] = []
```